### PR TITLE
Foundations - Global styles

### PR DESCRIPTION
**Pull request by Jasoneto**
HTML and CSS folders added, global styles added, typography added too

### CODE REVIEW

**Review by Theweeksad — APPROVED** [review]
NIce!!

**Comment by francoo-trainer** [comment]
nice!

**Review by johanmendezb — CHANGES_REQUESTED** [review]
good job, couple of comments:
1. Add global css styles to a global css folder.
2. Bring back the index.html to the root of the project.
3. Change html folder name to something for contextual to the project like "flow", "step".
4. Remove all extra code that is not related to the ticket.(no action needed)

**Review by Theweeksad — APPROVED** [review]
I see that the code is in good condition.

File: CSS/Question-01.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.content__title {
+    font-weight: 700;
+    font-size: 36px;
```
Comment (johanmendezb) [inline CSS/Question-01.css]: Line 2 and 3 should be located at global styles

File: CSS/Question-01.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+.content__subtitle {
+    font-size: 24px;
```
Comment (johanmendezb) [inline CSS/Question-01.css]: This should be located at global styles

File: CSS/Question-01.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+.btn__question-01 {
+    display: flex;
+    justify-content: center;
+    font-size: 16px;
```
Comment (johanmendezb) [inline CSS/Question-01.css]: `font-size` property is already 16px since it is the browser default, please remove this line

File: CSS/main.css [hunk 4]
```diff
@@ -1,7 +1,23 @@
-.heading__title {
-    font-family: 'Noto', serif;
+.content__title {
+    font-weight: 700;
+    font-size: 36px;
```
Comment (johanmendezb) [inline CSS/main.css]: Line 2 and 3 should be located at global styles

File: CSS/main.css [hunk 5]
```diff
@@ -1,7 +1,23 @@
...
 }
 
-.content__specifications {
-    font-family: 'Nunito';
-}
\ No newline at end of file
+.content__subtitle {
+    font-size: 24px;
+    font-weight: 400;
```
Comment (johanmendezb) [inline CSS/main.css]: Line 7 and 8 should be located at global styles

File: CSS/main.css [hunk 6]
```diff
@@ -1,7 +1,23 @@
...
+}
+
+.copy__specifications {
+    font-size: 16px;
```
Comment (johanmendezb) [inline CSS/main.css]: `font-size` property is already 16px since it is the browser default, please remove this line

File: CSS/main.css [hunk 7]
```diff
@@ -1,7 +1,23 @@
...
+    font-weight: 400;
```
Comment (johanmendezb) [inline CSS/main.css]: ` font-weight`property is already 400 as the default, please remove this line

File: CSS/main.css [hunk 8]
```diff
@@ -1,7 +1,23 @@
...
+    margin-bottom: 80px;
+    margin-top: 50px;
+}
+
+.btn__specifications {
+    display: flex;
+    font-size: 16px;
```
Comment (johanmendezb) [inline CSS/main.css]: font-size property is already 16px since it is the browser default, please remove this line

File: CSS/main.css [hunk 9]
```diff
@@ -1,7 +1,23 @@
...
+    font-weight: 400;
```
Comment (johanmendezb) [inline CSS/main.css]: font-weightproperty is already 400 as the default, please remove this line

**Review by johanmendezb — CHANGES_REQUESTED** [review]

**Review by Theweeksad — APPROVED** [review]
I see that everything is fine

**Comment by francoo-trainer** [comment]
thanks for the changes!

**Review by Theweeksad — APPROVED** [review]
all it's good, nice job

**Review by francoo-trainer — COMMENTED** [review]
good contribution, thanks

**Review by johanmendezb — APPROVED** [review]
GJ

**Review by francoo-trainer — APPROVED** [review]

**Review by Brayan1425 — APPROVED** [review]
Changes are good

**Pull request merged**